pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers=[
 requires-python = ">=3.9"
 dependencies = [
   "jsonschema>=4.4,<4.24",
-  "pulpcore>=3.68.2,<3.85",
```

Comment:
This is the "don't use unsupported branches as LBs" change. Nobody "requires" 3.70 - so this updates to current-latest, unless/until we call something out as "supported".